src/lib.rs

```diff
@@ -371,17 +371,55 @@ pub fn clamp<T: PartialOrd>(input: T, min: T, max: T) -> T {
     }
 }
 
+/// A value bounded by a minimum value
+///
+///  If input is less than min then this returns min.
+///  Otherwise this returns input.
+///  Preserves `NaN` different from `min`.
```

Comment:
I have trouble understanding "different from" here. Perhaps you mean "Preserves `NaN` regardless of `min`"?

Reply:
Fixed to: `clamp_min(std::f32::NAN, 1.0)` preserves `NAN` different from `f32::min(std::f32::NAN, 1.0)`

Reply:
Ah, I thought you were referring to the `min` parameter. It makes more sense now.